tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -1163,6 +1179,38 @@ bool IsMatchedMatMulBiasAddAndGeluExact(
   using utils::MatchingDirection;
   using utils::NodeStatus;
   // clang-format off
+  static utils::OpTypePattern* gelu_exact_pattern2 = new utils::OpTypePattern
```

Comment:
Nit: Pattern 2 should be after pattern 1. Please move this to below the definition of `gelu_exact_pattern`.